openedx/core/djangoapps/discussions/models.py

```diff
@@ -96,7 +96,7 @@ def __str__(self):
         )
 
     @property
-    def available_providers(self) -> List[str]:
+    def available_providers(self) -> list[str]:
```

Comment:
Is this necessary?

Reply:
Yes, this change is based on [pep585](https://www.python.org/dev/peps/pep-0585/) made by [pyupgrade typing-rewrites](https://github.com/asottile/pyupgrade#pep-604-typing-rewrites).